### PR TITLE
Features: Add Person, Student and Teacher classes

### PR DESCRIPTION
**Pull request by marurevi**
## OOP school library
- Create Person, Student, and Teacher classes
- Create Public and Private methods
- Create methods accessors

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+require_relative './person'
+
+# Student class
+class Student < Person
+  def initialize(classroom, age, name = 'Unknown', parent_permission: true)
+    super(classroom, age, name, parent_permission)
```
Comment (mikethreels) [inline student.rb]: - [x] classroom should not be included when calling super

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Mariana, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: student.rb (hunk 1 above)
Comment (marurevi) [reply student.rb]: Oh! you are right! I'll fix it 👍🏼

**Review by gemmen29 — APPROVED** [review]
Hi @marurevi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**